backend/src/org/akvo/lumen/auth.clj

```diff
@@ -33,13 +33,13 @@
 (defn wrap-jwt
   "Go get cert from Keycloak and feed it to wrap-jwt-claims. Keycloak url can
   be configured via the KEYCLOAK_URL env var."
-  [handler issuer]
+  [handler keycloak-url realm]
   (try
-    (let [certs (-> (str issuer "/realms/akvo/protocol/openid-connect/certs")
+    (let [issuer (str keycloak-url "/realms/" realm)
+          certs (-> (str issuer "/protocol/openid-connect/certs")
                     client/get
                     :body)]
       (jwt/wrap-jwt-claims handler (jwt/rsa-key certs 0) issuer))
     (catch Exception e
```

Comment:
@kardan Why do we need to catch, print and swallow exceptions here? Is there any point in continuing starting the server if we can't get the cert, especially since we'll return `nil` and not a handler?